project_task_project_mandatory/models/project_task.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    project_id = fields.Many2one(required=True)
```

Comment:
@victormartinelicocorp The `project_id` id defined in basic model project and project_issue. and it is depended by this module. So it is quite sure for the `project_id` exists in the table.